### DISCUSSION TITLE
Meeting Agenda 2/13

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#286] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Sprint Planning
3. Ask Darian about getting access to lab hardware for load testing (defer to next meeting with darian)
4. Discuss demo presentation on 2/15

**Comment by moreno-michael** [comment]
Meeting minutes 2/13
Virtual meeting

1. Task Review
- Update and complete all diagrams for Darian, commit all diagrams to spec doc when completed
- Issue #1239 complete pending approval, Issue #1295 in progress
- Issue #1296, #1297 in progress
- Issue #1206 in progress
2. Sprint Planning
- Sprint tasks will carry over from the previous week with tasks not completed by eod 2/13
- Issue #1286 reassigned to Carson
3. Classes cancelled so speaking to Darian about lab and hardware access will be pushed to next meeting
4. Demo discussion
- Make a short slide deck for presentation
- Ensure Issue #1239 (Docker DNS compatibility) is completed and ready for demo
- General idea of demo sequence discussed
